tests/xtro-sharpie/api-annotations-dotnet/common-GameKit.ignore

```diff
@@ -8,3 +8,10 @@
 
 ## unsorted
 
+!missing-field! GKSessionErrorDomain not bound
+!missing-selector! GKLeaderboardEntry::context not bound
+!missing-selector! GKLeaderboardEntry::date not bound
+!missing-selector! GKLeaderboardEntry::formattedScore not bound
+!missing-selector! GKLeaderboardEntry::rank not bound
+!missing-selector! GKLeaderboardEntry::score not bound
```

Comment:
Why are these ignored? Looks like they can be bound.

Reply:
@mandel-macaque tried binding them but still get similar intro failures as the ones mentioned here https://github.com/xamarin/maccore/issues/2269

as discussed in the thread, the GKLeaderboardEntry cannot be directly init, so i can try to figure out how to test them out somehow

Reply:
@mandel-macaque tried binding them but still get similar intro failures as the ones mentioned here https://github.com/xamarin/maccore/issues/2269

as discussed in the thread, the GKLeaderboardEntry cannot be directly init, so i can try to figure out how to test them out somehow